frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPaletteSearchInput/CommandPaletteSearchInput.module.css

```diff
@@ -6,13 +6,21 @@
 
 .inputContainer {
   display: flex;
+  width: 100%;
   margin-left: var(--spacing-2);
 }
 
+.inputWithSuggestion {
+  position: relative;
+  width: 100%;
+}
+
 .input {
+  position: relative;
   padding: var(--spacing-2) 0;
   width: 100%;
   border-radius: var(--border-radius-smbase);
+  z-index: 1;
```

Comment:
This change is to make the input element clickable when suffix displayed.
Without this change, the suffix (with `.completionSuffix` class) comes in front of the input because it has `position: absolute`.

<img width="803" height="130" alt="Image" src="https://github.com/user-attachments/assets/ed141085-86f1-4c23-8ca4-bab635c76958" />